reports/aspace2alma/get_MARCxml.rb

```diff
@@ -2,20 +2,38 @@
 require 'active_support/all'
 require 'nokogiri'
 require 'net/sftp'
+require 'byebug'
 require_relative '../../helper_methods.rb'
 
 #log errors to file
 $stderr.reopen("log_err.txt", "w")
 # #keep values synced so they're not going to the buffer
 $stderr.sync = true
 
+remote_filename = "sc_active_barcodes.csv"
+
 #configure sendoff to alma
+#rename old MARC file so we never send an outdated file by accident
 def alma_sftp (filename)
   Net::SFTP.start(ENV['SFTP_HOST'], ENV['SFTP_USERNAME'], { password: ENV['SFTP_PASSWORD'] }) do |sftp|
+    sftp.rename!(File.join('/alma/aspace/', File.basename(filename)), "/alma/aspace/MARC_out_old.xml")
```

Comment:
Do we want to use our rename method in the wrapper method, instead of within the `alma_sftp` method? Like we did with the other rename?